man/ocamlrun.m

```diff
@@ -69,22 +69,45 @@
 .B b
 flag in the OCAMLRUNPARAM environment variable (see below).
 .TP
+.B \-H
+Allocates heap chunks by mmapping huge pages. Huge pages are locked into
+memory, and are not swapped.
+.TP
```

Comment:
This is an `OCAMLRUNPARAM` setting `H`; there's no `ocamlrun` `-H` parameter (see function `parse_command_line` in `startup_byt.c`)